compiler/rustc_hir/src/lang_items.rs

```diff
@@ -331,6 +331,8 @@ language_item_table! {
     ResultOk,                sym::Ok,                  result_ok_variant,          Target::Variant;
     ResultErr,               sym::Err,                 result_err_variant,         Target::Variant;
 
+    SliceLen,                sym::slice_len_fn,        slice_len_fn,               Target::Method(MethodKind::Inherent);
```

Comment:
This should probably be moved to above the `// Language items from AST lowering`.